contrib/reprepro/suites.sh

```diff
@@ -5,7 +5,7 @@ cd "$(dirname "$BASH_SOURCE")/../.."
 
 targets_from() {
        git fetch -q https://github.com/docker/docker.git "$1"
-       git ls-tree -r --name-only "$(git rev-parse FETCH_HEAD)" contrib/builder/deb | grep '/Dockerfile$' | sed -r 's!^contrib/builder/deb/|-debootstrap|/Dockerfile$!!g'
+       git ls-tree -r --name-only "$(git rev-parse FETCH_HEAD)" contrib/builder/deb/${DOCKER_ENGINE_GOARCH:="amd64"} | grep '/Dockerfile$' | sed -r 's!^contrib/builder/deb/|-debootstrap|/Dockerfile$!!g'
```

Comment:
Shouldn't this actually use `*` (shell glob) instead of the arch of the engine we're currently running against to make sure we have the _full_ list of possible suites?